src/Pages/Profile/MemberView/InfoCard.js

```diff
@@ -11,93 +11,49 @@ export default function ProfileCard(props) {
   const [password, setPassword] = useState('New Password');
   const [confirmPass, setConfirmPass] = useState('Confirming New Password');
   const [user, setUser] = useState('');
+  const [state, setState] = useState({});
 
-  async function changePassword() {
-    // hash pass
-    const salt = bcrypt.genSaltSync(10);
-    const hashedPassword =
-      password.trim() === '' ? user.password : bcrypt.hashSync(password, salt);
-
-    if (password === confirmPass) {
-      const apiResponse = await editUser(
-        {
-          ...user,
-          password: hashedPassword
-        },
-        user.token
-      );
-      if (!apiResponse.error) {
-        setPassword('');
-        window.alert('Success!!');
-      }
-    }
-  }
-
-  function buttonStyle() {
-    let style = { marginTop: '10px' };
-    password === confirmPass
-      ? (style = {
-        ...style,
-        color: 'white'
-      })
-      : (style = {
-        ...style,
-        color: 'grey',
-        cursor: 'not-allowed'
+  function itemClicked(id) {
+    if (id === 5) {
```

Comment:
instead of doing `5`? can you assign 5 to a const variable like `printPageId`? it will help with readability